test/Microsoft.Framework.Logging.Test/FormattedLogValuesTest.cs

```diff
@@ -71,5 +72,80 @@ public void LogValues_With_UnbalancedBraces(string format)
                 logValues.ToString();
             });
         }
+
+        // message format, format arguments, expected message
+        public static TheoryData<string, object[], string> FormatsEnumerableValuesData
+        {
+            get
+            {
+                return new TheoryData<string, object[], string>
```

Comment:
What about test cases for:
1. The sub-enumerable has `null` values
2. Non-strings in the sub-enumerables